### PR TITLE
update webpack dependencies and config

### LINKED ISSUE #4: Also copy ThirdParty/Workers
@nmschulte brought up the following solution here: 

https://github.com/AnalyticalGraphicsInc/cesium/issues/5816#issuecomment-351201939
```
 new CopyWebpackPlugin([
        {from: path.join(paths.cesiumSource, 'Assets'), to: 'Assets'},
        {from: path.join(paths.cesiumSource, 'Widgets'), to: 'Widgets'},
        {from: path.join(paths.cesiumSource, 'ThirdParty'), to: 'ThirdParty'}
    ]), 
    new CopyWebpackPlugin([
        {from: path.join(paths.cesiumSource, '../Build/Cesium/ThirdParty/Workers'), to: 'ThirdParty/Workers'},
        {from: path.join(paths.cesiumSource, '../Build/Cesium/Workers'), to: 'Workers'}
    ])  
```

This update should be reflected in the tutorial as well.

### LINKED ISSUE #11: Base url issue
From https://github.com/AnalyticalGraphicsInc/cesium/issues/6319#issuecomment-410764699

> The documentation for Cesium and Webpack appears to be broken around 1.47, 1.48 . Whenever the iframe implementation was enforced as the dom container for the map.

---

> Heads up, I figured it out.
>
>`CESIUM_BASE_URL : JSON.stringify('')`
>now needs to be
>`CESIUM_BASE_URL : JSON.stringify('../')`

### LINKED ISSUE #24: webpack 5 example
Hi,

It would be useful with an example that works with webpack v5.

Regards

### PR DESCRIPTION
**Pull request by jjspace**
We are basically already inline with the correct ways to do webpack 5. This just updates the config slightly to demonstrate setting a static path and updates the related package versions.

I also added a `npm run start:built` to demonstrate hosting the built version as static files using `http-server`

This branches off of #51 

Fixes https://github.com/CesiumGS/cesium-webpack-example/issues/24
Fixes https://github.com/CesiumGS/cesium-webpack-example/issues/11
Fixes https://github.com/CesiumGS/cesium-webpack-example/issues/4

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -41,15 +44,31 @@ module.exports = {
...
+        },
+        {
+          from: path.join(cesiumSource, "Assets"),
+          to: `${cesiumBaseUrl}/Assets`,
+        },
+        {
+          from: path.join(cesiumSource, "Widgets"),
+          to: `${cesiumBaseUrl}/Widgets`,
+        },
+        // If you only installed @cesium/engine use these routes instead:
```
Comment (ggetz) [inline webpack.config.js]: IMO it would be cleaner to describe this in the README rather than inline comments.

Comment (jjspace) [reply webpack.config.js]: @ggetz I added a section to the readme. Let me know if that looks good

**Comment by ggetz** [comment]
Looks good to me! Thanks @jjspace!

File: webpack.config.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
 "use strict";
 
 // The path to the CesiumJS source code
```
Comment (nmschulte) [inline webpack.config.js:3]: This comment is out of place now.

Comment (jjspace) [reply webpack.config.js:3]: Will be fixed in a follow up pr

**Pull request merged**